avalon/tools/loader/lib.py

```diff
@@ -68,7 +68,7 @@ def add_representation_loaders_to_menu(loaders, menu):
     # List the available loaders
     for representation, loader in loaders:
         label = None
-        if representation:
+        if representation.get("custom_label"):
```

Comment:
This will break `SubsetLoader` where representation is `None`. Keep it as it was, the logic won't change... (label will still be None if `"custom_label"` is not on repreesntation.

Reply:
Fixed, I read changes when merging wrong